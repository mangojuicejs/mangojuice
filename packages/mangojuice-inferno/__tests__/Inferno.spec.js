import AbstractTests from 'mangojuice-react-core/tests';
import Subscribe from 'mangojuice-inferno';
import { Component } from 'inferno-component';
import { createElement } from 'inferno-create-element';
import { render } from 'inferno';


const React = { createElement, Component };
const ReactDOM = {
  render,
  unmountComponentAtNode: render.bind(null, createElement(() => {}))
};
AbstractTests(React, ReactDOM, Subscribe, 'Inferno');
