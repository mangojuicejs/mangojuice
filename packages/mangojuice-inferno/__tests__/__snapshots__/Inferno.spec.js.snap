// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`React interface view renderer: Inferno shuold render a stateless view of a block 1`] = `
Array [
  Object {
    "a": "test",
    "nested": null,
  },
  Array [
    Object {
      "a": "another",
    },
  ],
  Object {
    "a": "another",
  },
]
`;
